components/hero/Hero.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import styles from './hero.module.css'
+const Hero = () => {
+  return (
+    <section className ={styles.hero}>
+        <div className={styles.heroContent}>
```

Comment:
For consistency use space 2 for insets throughout the whole component (and other files you may created).